### PR TITLE
fix: text selection not disabled around game area

### LINKED ISSUE #26: 🐛 Bug: Text selection around the game should be disabled
Pressing/tapping and dragging triggers text selection 

image

### PR DESCRIPTION
**Pull request by ayo-run**
## Overview

What does this PR do? Why?
- disable user text selection around the game area
- prevent scenarios where text selection is triggered due to long-tapping buttons on mobile

### PR Checklist

-   [x] Fixes #26 
-   [x] I have run this code to verify it works
-   [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -57,6 +57,10 @@ section#game {
     min-height: 515px;
     margin-bottom: 35px;
     position: relative;
+    -ms-user-select: none;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    user-select: none;
```
Comment (JoshuaKGoldberg) [inline src/index.css]: https://caniuse.com/?search=user-select -- `user-select: none` has been supported by Chrome, Edge, and Firefox for a good amount of time now. It's just Safari that's a holdout. 

```suggestion
    -webkit-user-select: none;
    user-select: none;
```

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Looks very close to perfect! Just a small removal. Thanks! ❤️

**Comment by ayo-run** [comment]
Thanks for the review, applied the changes! 🙏

**Review by JoshuaKGoldberg — APPROVED** [review]
💯 🔥 ❗

**Pull request merged**